### DISCUSSION TITLE
Get-PnPTeamsTeam A parameter cannot be found that matches parameter name 'Filter'

### DISCUSSION THREAD

**Discussion by DevOp123** [pnp/powershell#2506] [Q&A] [answered]
Hi, 

I'm trying to use the -Filter parameter on the Get-PnPTeamsTeam cmdlet ( https://pnp.github.io/powershell/cmdlets/Get-PnPTeamsTeam.html ).  Example:  **_Get-PnPTeamsTeam -Filter "startswith(mailNickName, 'contoso')"_**

My exact code is: **_$Teams = Get-PnPTeamsTeam -Filter "startswith(mailNickName, 'finance-')"_**

However when I use this I get an error:  **A parameter cannot be found that matches parameter name 'Filter'.**

Please can you help?  is this a mistake on the documentation and Filter is not supported by this cmdlet??  thanks

**Comment by gautamdsheth** [comment, marked as answer]
Hi, this `-Filter` parameter is available in the latest nightly builds. It will also be available whenever we release 1.12.0 

To install nightly builds, you need to specify `-AllowPrerelease` parameter.

Check this link on how to do that:

https://www.easy365manager.com/install-module-a-parameter-cannot-be-found-that-matches-parameter-name-allowprerelease/
